estuary/utils/story.py

```diff
@@ -144,6 +145,59 @@ def get_sibling_nodes_count(self, results, reverse=False):
             return correlated_nodes[::-1]
         return correlated_nodes
 
+    def get_wait_times(self, results):
```

Comment:
Could you add a specific unit test for this method?

Reply:
After discussion, we decided that fixing the timestamps in `test_stories.py` would be sufficient.